src/main/java/com/google/sps/algorithms/FindMatchQuery.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.algorithms;
+
+import com.google.sps.data.MatchRequest;
+import com.google.sps.data.User;
+import java.util.ArrayList;
+import java.util.Collection;
+
+public final class FindMatchQuery {
```

Comment:
Class names are typically nouns - consider renaming this for clarity.

Reply:
renamed to MatchQuery!